Tests/ActiveLabelTests.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  ActiveLabelTests.swift
+//  UtiliKit-iOSTests
+//
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+import XCTest
+import UIKit
+@testable import UtiliKit
+
+class ActiveLabelTests: XCTestCase {
```

Comment:
Not sure how feasible this is, but I'd love to see some snapshot tests for this library (for the various configurations - like in the example gifs).

Reply:
Which library would you like me to use for those?

Reply:
The `iOSSnapshotTestCase` should be fine. I know there's a newer snapshotting library out there too, which I'm not opposed to using. Up to you!

Reply:
The point free snapshot testing library is glorious.

Reply:
Added Snapshot tests using the PointFree SnapshotTesting SDK.